assets/js/blocks/cart-checkout/cart-i2/block.js

```diff
@@ -1,37 +1,41 @@
 /**
  * External dependencies
  */
+import { __ } from '@wordpress/i18n';
 import { CART_STORE_KEY as storeKey } from '@woocommerce/block-data';
 import { dispatch } from '@wordpress/data';
 import { useStoreCart } from '@woocommerce/base-context/hooks';
-import { useEffect, RawHTML } from '@wordpress/element';
+import { useEffect } from '@wordpress/element';
 import LoadingMask from '@woocommerce/base-components/loading-mask';
 import { ValidationContextProvider } from '@woocommerce/base-context';
-import {
-	dispatchEvent,
-	translateJQueryEventToNative,
-} from '@woocommerce/base-utils';
+import { CURRENT_USER_IS_ADMIN } from '@woocommerce/settings';
+import BlockErrorBoundary from '@woocommerce/base-components/block-error-boundary';
+import { translateJQueryEventToNative } from '@woocommerce/base-utils';
 import withScrollToTop from '@woocommerce/base-hocs/with-scroll-to-top';
+import {
+	StoreNoticesProvider,
+	StoreSnackbarNoticesProvider,
+	CartProvider,
+} from '@woocommerce/base-context/providers';
+import { SlotFillProvider } from '@woocommerce/blocks-checkout';
 
-/**
- * Internal dependencies
- */
-import FullCart from './full-cart';
+const reloadPage = () => void window.location.reload( true );
 
-const EmptyCart = ( { content } ) => {
-	useEffect( () => {
-		dispatchEvent( 'wc-blocks_render_blocks_frontend', {
-			element: document.body.querySelector(
-				'.wp-block-woocommerce-cart'
-			),
-		} );
-	}, [] );
-	return <RawHTML>{ content }</RawHTML>;
-};
```

Comment:
We don't need RawHTML but we I think we should still fire this event.

Reply:
Is this being tracked?

Reply:
No, I will add it to this branch given I handled empty cart as well.